docs/configuration/identity-providers/oidc.md

```diff
@@ -138,39 +140,43 @@ identity_providers:
 ## Options
 
 ### hmac_secret
+
 <div markdown="1">
 type: string
 {: .label .label-config .label-purple } 
 required: yes
 {: .label .label-config .label-red }
 </div>
 
-The HMAC secret used to sign the [OpenID Connect] JWT's. The provided string is hashed to a SHA256 byte string for
-the purpose of meeting the required format.
-
-Can also be defined using a [secret](../secrets.md) which is the recommended for containerized deployments.
+The HMAC secret used to sign the [OpenID Connect] JWT's. The provided string is hashed to a SHA256
+byte string for the purpose of meeting the required format. It can also be defined using a
+[secret](../secrets.md) which is the recommended for containerized deployments. You must
+[generate this option yourself](#generating-options-yourself).
 
 ### issuer_private_key
+
 <div markdown="1">
 type: string
 {: .label .label-config .label-purple }
 required: yes
 {: .label .label-config .label-red }
 </div>
 
-The private key in DER base64 encoded PEM format used to encrypt the [OpenID Connect] JWT's. This can easily be
-generated using the Authelia binary using the following syntax:
-
-```console
-authelia rsa generate --dir /config
-```
+The private key in DER base64 encoded PEM format used to encrypt the [OpenID Connect] JWT's. Can also be
+defined using a [secret](../secrets.md) which is the recommended for containerized deployments. The reason
+for using only the private key here is that one is able to calculate the public key easily from the private
+key in this format (`openssl rsa -in rsa.key -pubout > rsa.pem`).
```

Comment:
Regarding this comment and Clements suggestion, maybe move it to the FAQ and link to it?

Reply:
Alright. I moved it to the FAQ.